docs/_static/wharf-ci-schema.json

```diff
@@ -0,0 +1,501 @@
+                        },
+                        "append-cert": {
+                            "type": "boolean"
+                        },
+                        "push": {
+                            "type": "boolean",
+                            "default": true
+                        },
+                        "args": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "examples": [
+                                [
+                                    "FIRST_ARG=Value from '.wharf-ci.yml' file!",
+                                    "SECOND_ARG=12453125"
+                                ]
+                            ]
+                        }
+                    }
+                },
+                "container": {
+                    "title": "Container build step type",
+                    "description": "Run commands in a container.",
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "image",
+                        "cmds"
+                    ],
+                    "properties": {
+                        "image": {
+                            "type": "string",
+                            "examples": [
+                                "alpine:latest"
+                            ]
+                        },
+                        "cmds": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "examples": [
+                                [
+                                    "echo 'hello world'"
+                                ]
+                            ]
+                        },
+                        "os": {
+                            "type": "string",
+                            "default": "linux",
+                            "enum": [
+                                "linux",
+                                "windows"
+                            ]
+                        },
+                        "shell": {
+                            "type": "string",
+                            "examples": [
+                                "/bin/sh"
+                            ]
+                        },
+                        "certificatesMountPath": {
+                            "type": "string",
+                            "examples": [
+                                "/usr/local/share/ca-certificates"
+                            ]
+                        },
+                        "secretName": {
+                            "type": "string",
+                            "examples": [
+                                "spark-system-tests-secret"
```

Comment:
Do we want to publish it?

Reply:
Changed to `myproject-system-tests-secret`